tools/codespell_ignore_words.txt

```diff
@@ -0,0 +1,2 @@
+atmost
+COO
```

Comment:
We also have `"--ignore-words-list", "ags,aray,asend,ba,classs,crate,falsy,feld,inflight,lits,nd,slowy,te,oint,conveniant"` in the `.pre-commit-config` file, presumably we don't want to have both an inline ignore list and a separate file.

Reply:
Ah I missed it somehow, so many different ways to configure these things ...

I pushed a commit moving the config to `pyproject.toml`. I guess one small advantage is that you can run `codespell` directly if you want and it will use the config rather than having to run `precommit run codespell --files <file>`. I can definitely revert it if you like to have everything in `precommit`

Reply:
> so many different ways to configure these things

Seriously.

> moving the config to pyproject.toml

Sounds good to me, I don't have a strong preference as long as we keep it consistent.